### ISSUE TITLE
layout does not position properly with multiple screens

### ISSUE THREAD

**Issue by tbarbugli** [jgallen23/layouts#2]
Hi,
I am using layout with multiple screen (13" and 24"). the size calc is still ok, but there is problem with the positioning of the window, I guess it does consider the screen position configuration as it is set in the screen prefs.

**Comment by jgallen23** [comment]
Can you give me a little more information?  How are the screens oriented, which commands does it mess up on?

**Comment by ghost** [comment]
I'm having the same issue.  On my 24", its positioning the screens as if they were on my 13".

**Comment by jgallen23** [comment]
Multiple screens at the same time?

If you are switching monitors, make sure you run `resize reset`.  That recalculates screen size.

If you are on mountain lion, they changed the way screens were stored, so thats a known bug

**Comment by ghost** [comment]
That doesn't fix it.  It still has the wrong size and the wrong placement on the bigger monitor.  I'm on Lion.

**Comment by jgallen23** [comment]
dual monitor or monitor plugged in as primary?

**Comment by ghost** [comment]
I have Dual monitor plugged in; configured as Big monitor above laptop with the Big monitor acting as the primary monitor.

**Comment by fuzzy76** [comment]
I'm running Mountain Lion with my MacBook Air closed and a 24" 1920x1200 external screen. Layouts seems to position windows like it was running on the internal display. Is this the ML bug, or something else?

**Comment by jgallen23** [comment]
Yeah, I'm having a lot of issues with getting the screen sizes of multiple monitors.  I might have to write an objective c wrapper because I can't get the other monitor dimensions reliably with applescript.

**Comment by fuzzy76** [comment]
https://github.com/brickattack/Alfred-Resizer manages to do it, but I have no idea what powers it.

**Comment by jgallen23** [comment]
Not really.  It always resizes to your primary monitor.

**Comment by fuzzy76** [comment]
Oh. Well, it's probably better than what layouts currently do, but not good enough I guess.

**Comment by zhaocai** [comment]
maybe you can take a look of ASObjC Runner, http://www.macosxautomation.com/applescript/apps/runner.html

**Comment by takuma7** [comment]
any updates on this?

**Comment by nbren12** [comment]
I am having this issue as well.
